student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+end
+
+student = Student.new('Science', 'John', 18)
+puts student.play_hooky
```

Comment:
- [ ] [OPTIONAL] Kindly remove the console logs in order to have a clean code base.